pre-pages/pgql-2.0-spec.md

```diff
@@ -2115,12 +2421,22 @@ For example:
 {% include image.html file="example_graphs/financial_transactions.png" %}
 
 ```sql
+--PGQL
 SELECT COUNT(e) AS num_hops
      , SUM(e.amount) AS total_amount
      , ARRAY_AGG(e.amount) AS amounts_along_path
 FROM MATCH ANY CHEAPEST (a:Account) (-[e:transaction]-> COST e.amount)* (b:Account)
        ON financial_transactions
 WHERE a.number = 10039 AND b.number = 2090
+--SQL
+SELECT * 
+FROM GRAPH_TABLE(financial_transactions 
+  MATCH (a IS Account) (-[e IS transaction]-> COST e.amount)* (b IS Account) 
+  KEEP ANY CHEAPEST 
```

Comment:
for now let's only have the PGQL variant for any queries containing CHEAPEST.

it's because on the standard side it will likely have a different syntax so we want to wait till it becomes clear. I guess we shouldn't have allowed CHEAPEST inside GRAPH_TABLE but back then we thought it was safe to assume but things have changed.

Reply:
so we essentially want to hide it as if CHEAPEST does not exist inside GRAPH_TABLE.. to prevent people from using it, until we figured out what the standard syntax will be and then we'll add it to PGQL and show the 'correct' GRAPH_TABLE version.